Convert2HF/README.md

```diff
@@ -0,0 +1,30 @@
+# Convert to HuggingFace
+This directory contains a script `convert_neox_pt_to_huggingface_neox.py` to convert PolyCoder checkpoints trained by [gpt-neox](https://github.com/EleutherAI/gpt-neox) into HuggingFace format, and a script `generate.py` to load the converted model and generate code from a given prompt.
+
+## Environment
+transformers 4.23.1
+
+## Convert
+The script `convert.sh` is an example of converting a 0.4B parameter model to HuggingFace format:
```

Comment:
How about this to emphasize how you can use the shell script. Note that this is contingent on making the model size a parameter, as suggested below.
```suggestion
You can use the `convert.sh` script to convert specified model to the HuggingFace format, using `./convert.sh 0-4B` (or pass a different model size). This script in turn invokes `convert_neox_pt_to_huggingface_neox.py`, which you can also call directly as follows:
```